src/content/api/merchants.mdx

```diff
@@ -0,0 +1,627 @@
+    List of assets accepted by the merchant. See [Accepted Asset Model](#accepted-asset-model).
+  </Property>
+
+  <Property name="location" type="location">
+    Physical location of the merchant
+  </Property>
+</Properties>
+
+---
+
+## Accepted Asset Model
+
+### Attributes
+
+<Properties>
+  <Property name="assetType" type="string">
+    The [type](/api/asset-types) of the asset
+  </Property>
+
+  <Property name="products" type="object">
+    List of [Products](#product-model) supported by the asset type. Only available for [epay.nzd](/api/assets#gift-cards) and [centrapay.token](/api/assets#token-model) asset types.
+  </Property>
+</Properties>
+
+---
+
+## Product Model
+
+### Attributes
+
+<Properties>
+  <Property name="id" type="string">
+    Product's unique identifier.
+  </Property>
+</Properties>
+
+---
+
+<Endpoint
+  method="POST"
+  path="/api/merchants"
+>
+  ## Create Merchant
+
+  This endpoint allows you to create a Merchant.
+
+  ### Attributes
+  <Properties>
+    <Property name="accountId" type="string" required>
+      Id of Merchant's owning Centrapay account.
+    </Property>
+
+    <Property name="name" type="string" required>
+      Merchant name.
+    </Property>
+
+    <Property name="country" type="string" required>
+      Merchant [ISO 3166](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) country code. Must match the "region" on the [Account](/api/accounts).
+    </Property>
+
+    <Property name="test" type="boolean" experimental required>
+      Flag indicating merchant is for testing.
+    </Property>
+
+    <Property name="settlementConfig" type="object" experimental>
+      Merchant [Settlement Config](#settlement-config-model).
+    </Property>
+
+    <Property name="location" type="location" experimental>
+      Physical Location of Merchant. It is highly recommended that you provide this otherwise, users won't be able to find you with our [Merchant Search](#search-merchants) API if they perform a origin + distance query.
+    </Property>
+  </Properties>
+
+  ### Errors
+  <Properties>
+    <Error code="403" message="BANK_ACCOUNT_MISMATCH">
+      The bank account in the settlement config does not belong to the same account.
+    </Error>
+
+    <Error code="403" message="ACCOUNT_REGION_MISMATCH">
+      The merchant's "country" does not match the "region" on the Account.
+    </Error>
+  </Properties>
+
+  <CodePanel slot="code-examples" title="Request" method="POST" path="/api/merchants">
+    ```bash
+    curl -X POST https://service.centrapay.com/api/merchants \
+      -H "X-Api-Key: $api_key" \
+      -H "Content-Type: application/json" \
+      -d '{
+        "accountId": "C4QnjXvj8At6SMsEN4LRi9",
+        "name": "Centrapay Cafe Auckland",
+        "country": "NZ",
+        "test": false
+      }'
+    ```
+  </CodePanel>
+
+  <CodePanel slot="code-examples" title="Response">
+    ```json
+    {
+      "id": "5ee0c486308f590260d9a07f",
+      "accountId": "yqwyya0rzz3vvshqw0474u89xtj5fn",
```

Comment:
```suggestion
      "accountId": "C4QnjXvj8At6SMsEN4LRi9",
```

Reply:
I know this is the existing documentation but the request and the response should really match ❤️